GVFS/GVFS.Service/Handlers/EnableAndAttachProjFSHandler.cs

```diff
@@ -87,9 +87,26 @@ public static bool TryEnablePrjFlt(ITracer tracer, out string error)
                 isNativeLibInstalled = ProjFSFilter.IsNativeLibInstalled(tracer, new PhysicalFileSystem());
                 if (!isNativeLibInstalled)
                 {
-                    string missingNativeLibMessage = "Native library is not installed";
-                    error = string.IsNullOrEmpty(error) ? missingNativeLibMessage : $"{error}. {missingNativeLibMessage}";
-                    tracer.RelatedError($"{nameof(TryEnablePrjFlt)}: {missingNativeLibMessage}");
+                    if (isRunning)
```

Comment:
Not part of your change, but as part of reviewing this I found the `isRunning` name ambigious - and what does it refer to. Looking up here, I see this refers to whether the service is running?

Reply:
Thanks for pointing this out, I will update the name to something more descriptive.

Reply:
@jamill, I renamed several of this variables let me know what you think.